platform/darwin/test/MLNImageSourceTests.m

```diff
@@ -39,4 +39,19 @@ - (void)testMLNImageSourceWithImage {
     XCTAssertNil(source.URL);
 }
 
+- (void)testSetURL {
```

Comment:
Nit:

```suggestion
- (void)testMLNImageSourceSetURL {
```

Reply:
Fixed in https://github.com/maplibre/maplibre-native/pull/2197/commits/8fdcaaad1008fe92772ae48170b7ae172c1bbe16